README.md

```diff
@@ -120,6 +120,7 @@ with optional overrides.
 - **additionalParameters** - (`object`) additional parameters that will be passed in the authorization request.
   Must be string values! E.g. setting `additionalParameters: { hello: 'world', foo: 'bar' }` would add
   `hello=world&foo=bar` to the authorization request.
+  - `use_app`: [Android only] specific the application package name that you'd like to use for authenticating. Otherwise, use webview as default.
```

Comment:
We can't use `additionalParameter` passing adding properties to the auth function. `additionalParameters` is is a key-value map of values to be passed to the auth request. This would have to be a top lever parameter (more like `dangerouslyAllowInsecureHttpRequests`).